lib/android/audio.nit

```diff
@@ -14,54 +14,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Android audio services
+# Android audio services, wraps a part of android audio API
 #
-# You can get a sound by loading it with a `SoundPool` or a `MediaPlayer`
-# the recommended way to load a sound is by it's resource ID or it's name
-# other ways are for advanced use
+# Use `load_sound`, `load_music`, `load_sound_from_res`
+# or `load_music_from_res` to get a `Sound`.
+# Also get a sound by loading it with a `SoundPool` or a `MediaPlayer`
```

Comment:
There are way too much informations to give that can be resumed in one short example in the module's commentary. Is it ok for you if I improve the mnit_simple example with more explanations and refers to it here ? Otherwise the short example will be somewhere around 30 lines with commentary in it if i want to be really clear about how to use the API ...